### PR TITLE
Bug 1633582 - Google provider labeling VM disks

### PR DESCRIPTION
**Pull request by milescrabill**
Bugzilla Bug: [1633582](https://bugzilla.mozilla.org/show_bug.cgi?id=1633582)

### CODE REVIEW

File: services/worker-manager/src/providers/google.js [hunk 1]
```diff
@@ -240,12 +245,13 @@ class GoogleProvider extends Provider {
             name: instanceName,
             labels: {
               ...cfg.labels || {},
-              'created-by': `taskcluster-wm-${this.providerId}`.replace(/[^a-zA-Z0-9-]/g, '-'),
-              'managed-by': 'taskcluster',
-              'worker-pool-id': workerPoolId.replace(/[^a-zA-Z0-9-]/g, '-').toLowerCase(),
-              'owner': workerPool.owner.replace(/[^a-zA-Z0-9-]/g, '-').toLowerCase(),
+              labels,
```
Comment (djmitche) [inline services/worker-manager/src/providers/google.js:248]: itym `...labels`?

File: services/worker-manager/src/providers/google.js [hunk 2]
```diff
@@ -240,12 +245,13 @@ class GoogleProvider extends Provider {
...
             },
             description: cfg.description || workerPool.description,
+            disks: {
+              ...cfg.disks || {},
+              labels,
```
Comment (djmitche) [inline services/worker-manager/src/providers/google.js:253]: (but this looks correct)

**Comment by milescrabill** [comment]
What would be a good way to test this?

**Comment by djmitche** [comment]
Ah, I thought there was already some testing for labels where these new labels could just be added to the list.  I don't see that for any of the providers, in fact..

**Comment by djmitche** [comment]
So I think this is fine as-is.

**Pull request merged**